datafusion/core/tests/dataframe/dataframe_functions.rs

```diff
@@ -394,6 +394,21 @@ async fn test_fn_approx_percentile_cont() -> Result<()> {
 
     assert_batches_eq!(expected, &batches);
 
+    // with number of centroids set
+    let expr = approx_percentile_cont(col("b"), lit(0.5), Some(lit(2)));
```

Comment:
NOTE: I don't have SQL Server, so I did not verify that it produces this result.

I chose a choice for `centroids` that produced a different result than the default.